source/onboarding/elasticsearch/index.html.md.erb

```diff
@@ -0,0 +1,21 @@
+---
+title: ElasticSearch
+last_reviewed_on: 2020-03-18
+review_in: 6 months
+weight: 2
+---
+
+# <%= current_page.data.title %>
+
+We have implemented a fix to ElasticSearch to give us more reliability and integrity that the cases in ES will match those in the CCD-Data-Store database.
+
+
+
+At the time of writing Probate, CMC, SSCS, Employment, Divorce and FPLA are the only approved consumers of production ElasticSearch, we understand that this may change in the future as teams will want to utilise the enhanced functionality and performance that ElasticSearch provides. The following process must be adhered to for teams wanting to consume ElasticSearch in production
+
+
+1. Contact CCD and talk through your requirements with them explaining your circumstances and why you want to use ES in prod
```

Comment:
how who where?

Reply:
1. Contact CCD on Slack (#ccd-questions-answers) and talk through your requirements with them explaining your circumstances and why you want to use ES in prod